<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAccountFormClass</class>
 <widget class="QWidget" name="AddAccountFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddAccountForm</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="uinLabel">
     <property name="text">
      <string>UIN</string>
     </property>
     <property name="buddy">
      <cstring>uinEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="passwordLabel">
     <property name="text">
      <string>Password</string>
     </property>
     <property name="buddy">
      <cstring>passwordEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="passwordBox">
     <property name="text">
      <string>Save password</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="uinEdit"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="passwordEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>uinEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>passwordBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
